android/java/org/chromium/chrome/browser/crypto_wallet/util/AndroidUtils.java

```diff
@@ -43,4 +47,37 @@ public static Spanned formatHTML(String html) {
             return Html.fromHtml(html);
         }
     }
+
+    // Views
+    public static TextView makeHeaderTv(Context context) {
```

Comment:
why do we call the function as general `makeHeaderTv` if it's related to solana only. `AndroidUtils.java` contains general utils helper functions for the whole crypto wallet. We should call it somehow `makeSolanaHeaderTv` if it's relate to Solana only or pass the string as an argument, so it could be used everywhere.

Reply:
It's just a general function to create a header textview with a bold style. Don't wanna pass a string as it could be null etc. so don't wanna make it more specific than creating a header textview. we can make more helper function in the future if it is required.

Reply:
so why do we do `textView.setText(R.string.brave_wallet_solana_max_retries);` if it's just to create a header textview, why not some other string. We should remove `setText` if it's just a helper function to create a header textview with a bold style or move it out of utils if it's only to show that particular string and we cannot reuse it for other strings

Reply:
missed to remove that `setText` call, causing the confusion, will remove.